reana_server/scheduler.py

```diff
@@ -145,7 +146,7 @@ def requeue_workflow(self, **kwargs):
                 priority=kwargs.get("priority", 0),
                 min_job_memory=kwargs.get("min_job_memory", 0),
                 retry_count=kwargs.get("retry_count", 0) + 1,
-                delay=REANA_SCHEDULER_SECONDS_RETRY_DELAY * 1000,
+                delay=0,
```

Comment:
If we don't need delays I guess we need to partially revert this commit? https://github.com/reanahub/reana-commons/commit/65836cdb3e9c7d23c15e2715dcd3cc75f9690c3c